test/e2e/cronjob.go

```diff
@@ -225,10 +235,227 @@ var _ = framework.KubeDescribe("CronJob", func() {
 		err = deleteCronJob(f.ClientSet, f.Namespace.Name, cronJob.Name)
 		Expect(err).NotTo(HaveOccurred())
 	})
+
+	// cleanup of successful finished jobs, with default limits
+	It("should delete successful finished jobs according to default history limits [Slow]", func() {
```

Comment:
Make sure the test is fast, iow. instead of `sleep XXX` let it just print hello world, or alike.